packages/component-library/src/components/form/sw-button/sw-button.vue

```diff
@@ -7,6 +7,7 @@
     class="sw-button"
     :class="buttonClasses"
     v-bind="$attrs"
+    role="button"
```

Comment:
This is an accessibility violation. It may look like a button but it works like a link, because it is one.

Reply:
You're correct. I added it because some tests were failing because they are looking for a button. But I will adjust the tests then